packages/alice_objectbox/lib/alice_store.dart

```diff
@@ -0,0 +1,38 @@
+import 'dart:io' show Directory;
+
+import 'package:alice_objectbox/model/cached_alice_http_call.dart';
+import 'package:path_provider/path_provider.dart';
+import 'package:path/path.dart' as path;
+
+import 'objectbox.g.dart';
+
+/// Provides access to the ObjectBox Store throughout the app.
+///
+/// Create this in the apps main function.
+class AliceStore {
+  AliceStore._create(this._store) {
+    httpCalls = Box<CachedAliceHttpCall>(_store);
+  }
+
+  /// Create an instance of ObjectBox to use throughout the app.
+  static Future<AliceStore> create([Store? store]) async {
+    final String storeDirectoryPath = path.join(
+      (await getApplicationDocumentsDirectory()).path,
```

Comment:
What if this path will be unavailable? We should throw error.

Reply:
That's handled by
```dart
/// Creates the directory if it doesn't exist.
await Directory(storeDirectoryPath).create(recursive: true);
```

Reply:
Added a more explicit `Directory.exists` check
```dart
final Directory storeDirectory = Directory(storeDirectoryPath);
if (!await storeDirectory.exists()) {
  await storeDirectory.create(recursive: true);
}
```